README.adoc

```diff
@@ -1126,6 +1126,12 @@ $ just --set os bsd
 
 === Environment Variables
 
+==== How to pass environment variables to recipes
+
+There are two ways to pass environment variables to recipes: exporting `just` variables, or using an `.env` file.
+
```

Comment:
I think this section is probably unnecessary

```suggestion
```

Reply:
I removed it